AppConnectSwift/CaptureImageViewController.swift

```diff
@@ -77,14 +111,44 @@ class CaptureImageViewController: UIViewController, UIImagePickerControllerDeleg
             presentViewController(imagePicker, animated: true, completion: {})
         }
     }
+    
+    func loadSubjects() {
+        var bgQueue : NSOperationQueue! = NSOperationQueue()
+        bgQueue.addOperationWithBlock() {
+            let clientFactory = MDClientFactory.sharedInstance()
+            let client = clientFactory.clientOfType(MDClientType.Network);
+            var datastore = MDDatastoreFactory.create()
+            let user = datastore.userWithID(self.userID)
+            
+            // Start an asynchronous task to load the subjects for the user logged in
+            client.loadSubjectsForUser(user) { (subjects: [AnyObject]!, error: NSError!) -> Void in
```

Comment:
The loadsubjects method in mdsubject.h just returns an nsarray, with no specifier, so I think this has to be AnyObject.